datadog_checks_dev/datadog_checks/dev/tooling/commands/create.py

```diff
@@ -176,6 +182,9 @@ def create(ctx, name, integration_type, location, non_interactive, quiet, dry_ru
     for file in files:
         file.write()
 
+    if manifest_v2:
```

Comment:
Is there a reason why we first create a manifest v1 then migrate it, instead of creating the manifest v2 first?

Reply:
Mainly to simplify the code for the initial setup and templates, but continue to rely on the migration code.  This means as we add improvements to the migration process (if there are any) it will just carry over to the create process too.